src/microbe_stage/MicrobeAI.cs

```diff
@@ -740,7 +740,11 @@ private void LaunchToxin(Vector3 target)
             if (CanShootToxin())
             {
                 microbe.LookAtPoint = target;
-                microbe.QueueEmitToxin(oxytoxy);
+
+                // hold fire until the target is lined up.
+                if (microbe.FacingDirection().Normalized().AngleTo(microbe.LookAtPoint.Normalized()) <
+                    0.1f + SpeciesActivity / (Constants.AI_BASE_TOXIN_SHOOT_ANGLE_PRECISION * SpeciesFocus))
+                    microbe.QueueEmitToxin(oxytoxy);
```

Comment:
```suggestion
                // Hold fire until the target is lined up.
                if (microbe.FacingDirection().Normalized().AngleTo(microbe.LookAtPoint.Normalized()) <
                    0.1f + SpeciesActivity / (Constants.AI_BASE_TOXIN_SHOOT_ANGLE_PRECISION * SpeciesFocus))
                 {
                    microbe.QueueEmitToxin(oxytoxy);
                 }
```
I add braces when there are multiline if conditions to make it visually easier to see what's inside the if statement and what is part of the condition.

Reply:
That's also my preference. I must have missed that.

Reply:
Let's hope I didn't make a mistake with the GitHub editor... If this passes checks then this is fine for merging as long as no one else wants to playtest this.